packages/client-sdk-web/src/cache-client.ts

```diff
@@ -1,13 +1,12 @@
 import {ControlClient} from './internal/control-client';
 import {DataClient} from './internal/data-client';
+import {NoopMomentoLoggerFactory, CredentialProvider} from '@gomomento/core';
 import {
   AbstractCacheClient,
   IControlClient,
-  NoopMomentoLoggerFactory,
-  CredentialProvider,
   IDataClient,
   ICacheClient,
-} from '@gomomento/common';
+} from '@gomomento/core/dist/src/internal/clients/index';
```

Comment:
I thought we didn't want to have `/dist/` imports?

Reply:
discussed on the call - we don't want **users** to ever need to do imports from `dist`; in this case these are private internal classes that users shouldn't be using, so it's fine.